FundacionParaguaya/MentorApp
# Rollover fixes all over the app

There are 3 major issue being fixed with this pull request:
- Interactive elements (buttons/icons) will no longer fade out when tapped, but will increase their color intensity. For example `palegreen` will go to `green`.
- List items like Drafts and Surveys will have a background active color
- Stoplight Questions will show the Check circle icon when tapped

**To Test**
1. Go to Dashboard
2. Hold tap on the Create a life map button, without letting go. It's background should become darker (`colors.green`).
3. Hold tap on the burger menu. It's color should again be darkened.
4. Open the menu and repeat the same steps for the links and logout button.
5. Open logout modal. Hold tapping on the Yes button should make it darker red. The now should be a darker grey.
6. Create a new lifemap. In the choose survey screen, hold tap on any survey item from the list. It should have a grey background instead of fading out.
7. Go to Primary participant. Open any select. The options should have a grey background on hold press.
8. Go to the stoplight questions. Hold tapping an indicator should show the circle thick icon with the appropriate color (as if already having a value).
9. Go through the lifemap skipping a few questions
10. Go to skipped questions. Hold tapping a question should produce the same effect as with surveys on step 6.
11. Go to overview and try adding a priority. Tapping the number of months (counter elements) should make them green background on white + or -.
12. From Overview screen open the open filters and check if again the same effect as with surveys from step 6 is present when tapping.

@IvaKop This pull request also includes the overview filters list item being separated into its own component. We discussed that I will do this without an issue.
